lib/mail-archive-helper.ts

```diff
@@ -157,6 +157,9 @@ export class MailArchiveGitHelper {
                         .addPRComment(pullRequestURL, comment);
                 }
 
+                await this.githubGlue.addPRcc(pullRequestURL,
+                                              parsedMbox.from || "");
```

Comment:
Maybe we should guard this more like this:

```js
        if (parsedMbox.from) {
                await this.githubGlue.addPRcc(pullRequestURL,
                                              parsedMbox.from);
        }
```

But then, we return without doing anything from `addPRcc()` if no email address was in that value, so I'd be fine with leaving this as-is.

Reply:
The chance of an email without a `from:` seems pretty low so will leave the code as is.